Update and rename GITHUB_TOKEN_SETUP

ezequiel-rodriguez opened the pull request:
PAT is no longer used, I updated to set up the GH App on the sources repositories for the changelog or other sections PRs for the DevPortal

bguiz commented on rif/rns/try-rns.md:
```diff
@@ -3,6 +3,12 @@ layout: rsk
 ---
 
-More information coming soon!
+Welcome to RNS! You can use RSK testnet to try the service for free.
```
```suggestion
Welcome to RNS! You can use the RSK Testnet to try the service for free.
```

bguiz commented on rif/rns/try-rns.md:
```diff
@@ -3,6 +3,12 @@ layout: rsk
+Welcome to RNS! You can use RSK testnet to try the service for free.
 
-[Collaborate on Github](https://github.com/rsksmart/rsksmart.github.io/issues/208)
+1. Download a browser wallet. Recommended: [Nifty Wallet](https://chrome.google.com/webstore/detail/nifty-wallet/jbdaocneiiinmjbjlgalhcelgbejmnid)
+2. Connect your wallet to RSK Testnet. You can do it with the top left selector on Nifty Wallet.
+3. (optional) List tRIF token balance using this address: 0x19F64674D8A5B4E652319F5e239eFd3bc969A1fE. In Nifty wallet do this in Tokens tab.
+4. Browse to [RSK faucet](https://faucet.testnet.rsk.co) to get some gas.
+5. Browse to [tRIF faucet](https://faucet.rifos.org) to get some test RIF tokens.
+6. Browse to [RNS Testnet Manager](https://testnet.manager.rns.rifos.org) and search for your desired domain.
+7. Follow the registration process.
```
```suggestion
1. Download a browser wallet. Recommended: [Nifty Wallet](https://chrome.google.com/webstore/detail/nifty-wallet/jbdaocneiiinmjbjlgalhcelgbejmnid)
2. Connect your wallet to the RSK Testnet. You can do it with the top left selector on Nifty Wallet.
3. (optional) List tRIF token balance using this address: `0x19F64674D8A5B4E652319F5e239eFd3bc969A1fE`. In Nifty wallet do this in the Tokens tab.
4. Use the [RSK faucet](https://faucet.testnet.rsk.co) to get some tR-BTC for gas.
5. Use the [tRIF faucet](https://faucet.rifos.org) to get some tRIF tokens.
6. Browse to the [Testnet RNS Manager](https://testnet.manager.rns.rifos.org) and search for your desired domain.
7. Follow the registration process.
```

ezequiel-rodriguez committed:
Update and rename GITHUB_TOKEN_SETUP 

PAT is no longer used, I updated to set up the GH App on the sources repositories for the changelog or other sections PRs for the DevPortal

owans commented on GITHUB_APP_SETUP.md:
```diff
@@ -0,0 +1,188 @@
+Everything is done through the GitHub App.
+
+## Prerequisites
+The GitHub App must be installed in rsksmart and have access to:
```
This process is handled internally

ezequiel-rodriguez committed:
Update GITHUB_APP_SETUP.md

owans reviewed (approved):
LGTM!
